### DISCUSSION TITLE
There are many missing translations in Simplified Chinese

### DISCUSSION THREAD

**Discussion by x1ntt** [LycheeOrg/Lychee#2535] [Q&A] [answered]
I noticed that there are many missing Chinese translations in the new version, so I plan to submit some translations. However, I have noticed some issues with `lang/zh_CN/lyche.php`.
+ Many items are no longer in use and I couldn`t find them in the app (so just ignore them?)
+ Some strings in the application are not in this file, such as` Set album thumbs aspect ratio `. In this case, if I want to continue localizing, I may need to refer to it` https://github.com/LycheeOrg/Lychee/pull/1270 `Modify the string in the code to something similar to `<h1>{$local ['Photo_SBOUT']}</h1>`?
(Regarding this, perhaps you should update it https://lycheeorg.github.io/docs/contributions.html#our-coding-style ,By the way, add some translation methods）
I am not very good at PHP, but if you are willing, I can try modifying the code in the project to achieve localization. Otherwise, I will only modify the existing localized text until someone changes those strings into variables.

In addition, as this is my first attempt to submit code, I would like to confirm the `pr` process, which is: `fork` project to my account **->**create sub branches from `master` **->**modify on sub branches **->**submit `pr` to the source project `master`

**Comment by ildyria** [comment, marked as answer]
> Many items are no longer in use and I couldn`t find them in the app (so just ignore them?)

Some of them are still used in [Lychee-front](https://github.com/LycheeOrg/Lychee-front) which is the legacy/nothing-works-solution.

> Some strings in the application are not in this file [...]  I am not very good at PHP, but if you are willing, I can try modifying the code in the project to achieve localization. Otherwise, I will only modify the existing localized text until someone changes those strings into variables.

I am currently in the middle of the rewrite of the front-end.
See:
- https://github.com/LycheeOrg/Lychee/tree/v6 branch on main repository
- https://lycheeorg.github.io/2024-06-29-future-of-lychee/

So I would suggest otherwise as I cannot guarantee I would be using those strings in the new front-end and modifying the Livewire front-end is kind of a waste of time.

> In addition, as this is my first attempt to submit code, I would like to confirm the pr process, which is: fork project to my account **->**create sub branches from master **->**modify on sub branches **->**submit pr to the source project master

Yes. :)

**Reply by x1ntt** [reply]
I am looking forward to the availability of v6. Although you mentioned three months ago that you were unsure about the launch time, I still want to know the launch time of v6 version

In addition, simply modifying the strings in lychee. php will not consume too much time, so I want to continue doing this (the 5.5.1 version I am currently using has many strings that are not in Chinese, and simply modifying the lychee. php file can improve a lot). Of course, I will only continue with it if you allow me

Thank you for your work ❤

**Reply by ildyria** [reply]
If you modify anything in the lang folder, I can guarantee you that the changes will be visible in v6 directly. Over the last few versions we have always used those and that is not going to change. 

> I still want to know the launch time of v6 version.

Hard to say, I really wish I could say by the end of the month, but even that is difficult to predict.
That being said, there is  now a very alpha version available as a docker tag. It is in its premature phase, there are a lot of things missing, but at least it is taking some shape. If you want to see the alpha in action, you can check my personal website:
https://alpha.photography.viguier.nl/ and if you want to compare with the current v5, just remove the alpha in the url.

**Reply by x1ntt** [reply]
V6 looks great, looking forward to an improvement in loading speed

Thank you for your patient reply. Have a pleasant life~ 🫡
